tests/test_automation/github-actions/ci/run_step.sh

```diff
@@ -375,7 +383,7 @@ case "$1" in
 
     if [[ "${GH_JOBNAME}" =~ (AFQMC-Offload) ]]
     then
-       export LD_LIBRARY_PATH=/opt/llvm/15.0.0-rc3/lib:/usr/lib64/openmpi/lib:${LD_LIBRARY_PATH}
+       export LD_LIBRARY_PATH=/opt/llvm/15.0.0/lib:/usr/lib64/openmpi/lib:${LD_LIBRARY_PATH}
```

Comment:
Does not need fixing here, but clearly this openmpi setting is in the wrong location; there should be an "if MPI" section.

Reply:
If I recall correctly AFQMC always needs MPI that was the logic behind this if. We can change it.